airbyte-integrations/connectors/source-tyntec-sms/source_tyntec_sms/spec.yaml

```diff
@@ -0,0 +1,21 @@
+documentationUrl: https://docs.airbyte.com/integrations/sources/tyntec-sms
+connectionSpecification:
+  $schema: http://json-schema.org/draft-07/schema#
+  title: Tyntec Sms Spec
+  type: object
+  required:
+    - api_key
+    - request_id
+  additionalProperties: true
+  properties:
+    api_key:
+      type: string
+      title: Tyntec API Key
+      description: Your Tyntec API Key. See <a href="https://www.tyntec.com/docs/docs-center-sms-api-quick-start">here</a>
+      order: 0
+      airbyte_secret: true
+    request_id:
+      type: string
+      title: SMS Message Request ID
+      description: The unique identifier provided for each SMS messaging request.
+      order: 1
```

Comment:
I think this should be removed and create a new stream to retrieve all requests id/messages and use a stream slices with the message stream.

Reply:
@marcosmarxm The new stream you suggested to retrieve all request IDs/messages will likely require the addition of new fields `to`, `from`, and `message`, which are required for the [linked endpoint](https://api.tyntec.com/reference/sms/current.html#sms-api-Send%20SMS%20(GET)). Is this acceptable?

Reply:
You can implement `to` `from` but check my previous comment.